### PR TITLE
New profile test

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: components/block_v2/stylesheets/index.styl [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.Block
+  flex 1 1 315px
+  text-align center
+  margin 0 10px 90px 10px
+  max-width 315px
```
Comment (dzucconi) [inline components/block_v2/stylesheets/index.styl]: `315px` should be extracted and named something

Comment (broskoski) [inline components/block_v2/stylesheets/index.styl]: Yes will do this

File: components/block_v2/stylesheets/index.styl [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    
+    &--text
+      border 1px solid bordergray
+      display block
+      padding unit
+      overflow hidden
+      font-family font_serif 
+      text-align left
+      p
+        p()
```
Comment (dzucconi) [inline components/block_v2/stylesheets/index.styl]: Do these mixins actually correspond with out own font sizes and so forth defined in the variables file? (Of which there are already too many).

File: components/block_v2/templates/block.jade [hunk 3]
```diff
@@ -0,0 +1,20 @@
+- extraClass = block.kind.__typename === 'Channel' ? "Block__inner--channel--" + block.kind.visibility : ''
```
Comment (dzucconi) [inline components/block_v2/templates/block.jade]: I don't get this whole `extraClass` thing. Name it something meaningful or inline it or extract it to a helper.

**Comment by dzucconi** [comment]
Lookin' goooood

**Comment by broskoski** [comment]
Alrighty, when deployed this is just going to force admins to view this version of the profile over the current one.

**Pull request merged**